src/main/java/mat/client/cqlworkspace/CQLMeasureWorkSpacePresenter.java

```diff
@@ -1912,7 +1932,14 @@ protected String getCurrentModelType(){
 		return MatContext.get().getCurrentMeasureModel();
 	}
 
+	protected boolean isFhirModelType() {
+		boolean result = !"QDM".equals(getCurrentModelType());
```

Comment:
Legacy pre-QDM measures can have model type  null/empty

Reply:
Good catch, will change that to "FHIR".equals instead of !"QDM".equals so it handles all those cases too.

Reply:
Changed it to use the MatContext.get().isFhirModelType method I added that uses "FHIR".equals and removed that method in wspresenter.